openqabot/openqa.py

```diff
@@ -105,3 +105,33 @@ def is_devel_group(self, groupid: int) -> bool:
         return (
             ret[0]["parent_id"] == DEVELOPMENT_PARENT_GROUP_ID if ret else True
         )  # ID of Development Group
+
+    @lru_cache(maxsize=256)
+    def get_single_job(self, job_id: int):
+        ret = None
+        try:
+            ret = self.openqa.openqa_request(
+                "GET",
+                "jobs/%s" % job_id,
+            )["job"]
+        except Exception as e:
```

Comment:
why do we need a broad exception here?

Reply:
Again, because it was done all over the place already and I kept it consistent. Will remove it as requested.

Reply:
https://github.com/openSUSE/qem-bot/blob/master/openqabot/openqa.py#L50
https://github.com/openSUSE/qem-bot/blob/master/openqabot/openqa.py#L73
https://github.com/openSUSE/qem-bot/blob/master/openqabot/openqa.py#L86
https://github.com/openSUSE/qem-bot/blob/master/openqabot/openqa.py#L100